### PR TITLE
Nightly release auto nvbug

### PR DESCRIPTION
**Pull request by albert17**
- Need to figure out what to add in points 5.
- Need to set proper bug configuration (ARB, CC, Type, Status, ...)

### CODE REVIEW

File: ci/nightly.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+args = parser.parse_args()
+
+# Set Table
+mydata = [
+    ["1", "NGC catalog URL of the container", args.cont],
+    ["2", "Version published to the NGC catalog", "nightly"],
+    ["3", "Expected publishing date and time", "COMPLETED"],
+    ["4", "QA results", args.qa],
+    ["5", "Security scan results", "??"],
+    ["6", "SWIPAT Approval", "??"],
```
Comment (jperez999) [inline ci/nightly.py:27]: the following link is the nvbug that gives us SWIPAT approval, so you can add it in: https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=3267056&cmtNo=

**Comment by albert17** [comment]
@jperez999

Added SWIPAT link.
Changed Table from txt to html.
Added automated Bug Creation.

Parameters allow to pass container name and log output.

Need to set proper bug configuration

**Comment by benfred** [comment]
@albert17 @jperez999 can we close this?

**Comment by albert17** [comment]
@benfred @jperez999 Yes we can. I already discussed with Julio that it is better to get this out of here.

**Pull request closed without merging**